audio_transcription.py

```diff
@@ -5,46 +5,47 @@
 
 class DialogueTranscription():
 
-    def __init__(self, path):
-        self.path = path
-        self.recognizer = sr.Recognizer()
-        self.transcription = ""
+    def __init__(self, audio_path):
+        self.audio_path = audio_path
+        self.text_recognizer = sr.Recognizer()
+        self.dialogue_transcription = ""
 
-    def transcribe(self,filename):
-        with sr.AudioFile(filename) as source:
-            audio_listened = self.recognizer.record(source)
-            text = self.recognizer.recognize_google(audio_listened)
-        return text
+    def transcribe(self, audio_file_name):
```

Comment:
As we are improving these code, I got one more improvement suggestion 

```suggestion
    def transcribe(self, audio_file_name : str) -> str:
```

As python language does not support strong typing, it's good to make such suggestion.
PS: Applicable also for other methods